ExternalModule.php

```diff
@@ -0,0 +1,87 @@
+<?php
+
+namespace HideChoiceByEvent\ExternalModule;
+
+use ExternalModules\AbstractExternalModule;
+use ExternalModules\ExternalModules;
+
+
+/**
+ * ExternalModule class for Project Ownership module.
+ */
+class ExternalModule extends AbstractExternalModule {
+
+    /**
+     * @inheritdoc
+     */
+    function hook_every_page_top($project_id) {
+        if (PAGE == 'Design/online_designer.php' && $project_id) {
+            $this->includeJs('js/modify_help_menu.js');
+            return;
+        }
+
+        //if not on a data entry page or survey, then don't do anything
+        if (!in_array(PAGE, array('DataEntry/index.php', 'surveys/index.php'))) {
+            return;
+        }
+
+        global $Proj;
+        $settings = array();
+
+        //get the name of every field on this page.
+        $fields = array_keys($Proj->forms[$_GET['page']]['fields']);
+
+        foreach($fields as $field) {
+            $field_info = $Proj->metadata[$field];
+
+            //continue if the field does not have any action_tags at all.
+            if (!$action_tags = $field_info['misc']) {
```

Comment:
The regex won't work for json containing line breaks. To fix that, I suggest replacing this line with:
```
if (!$action_tags = str_replace("\n", '', $field_info['misc'])) {
```